sdk/logs/src/main/java/io/opentelemetry/sdk/logs/ExtendedSdkLogger.java

```diff
@@ -24,7 +26,7 @@ final class ExtendedSdkLogger extends SdkLogger implements ExtendedLogger {
   }
 
   @Override
-  public boolean isEnabled() {
+  public boolean isEnabled(Severity severity, Context context) {
     return loggerEnabled;
```

Comment:
Unfortunately there's no way for the java SDK to currently respond to context or severity parameters today until we also implement LogRecordProcessor#isEnabled: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/sdk.md#logrecordprocessor-operations